configs/katello/3.15.yaml

```diff
@@ -0,0 +1,57 @@
+---
+:project: katello
+:github_org: katello
+:prior_releases:
+:releases:
+  :3.15.0:
+    :redmine_version_id: 1131
+:code_name: Rocky Road
+:repos:
+  :katello:
+    :main: true
+    :branch: KATELLO-3.15
+    :repo: https://github.com/Katello/katello.git
+    :version_branch: true
+  :foreman-packaging:
+    :branch: rpm/1.24
```

Comment:
Foreman is at 2.0 for this release so the branch will be `rpm/2.0`, but confirm that in foreman-packaging